File: src/net/eduard/economy/command/EconomySetCommand.java

package net.eduard.economy.command;

import net.eduard.economy.EduEconomy;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.lib.modules.Extra;
import net.eduard.api.lib.game.FakePlayer;

public class EconomySetCommand extends CommandManager {

	public EconomySetCommand() {
		super("set", "setar", "define", "difinir");
		setUsage("/money set <jogador> <quantidade>");
		setDescription("Definir saldo para o jogador");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		if (args.length < 3) {
			sendUsage(sender);
		} else {
			FakePlayer fakeplayer = new FakePlayer(args[1]);
			Double valor = Extra.fromMoneyToDouble(args[2]);
			valor = Math.abs(valor);
			EduEconomy.getInstance().getManager().setBalance(fakeplayer, valor);
			if (args.length >= 4){
				if (args[3] ==("-msg")){
					return true;
				}
			}
			sender.sendMessage(EduEconomy.getInstance().message("money-set").replace("$player", fakeplayer.getName())
					.replace("$amount", Extra.formatMoney(valor)));
			if (fakeplayer.getPlayer() != null) {
				fakeplayer.getPlayer()
						.sendMessage(EduEconomy.getInstance().message("money-changed").replace("$player", sender.getName()));
			}
		}

		return true;
	}

}

File: src/main/java/net/eduard/economy/command/EconomySetCommand.kt
package net.eduard.economy.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.FakePlayer
import net.eduard.api.lib.modules.Extra
import net.eduard.economy.EduEconomyPlugin
import org.bukkit.command.CommandSender
import kotlin.math.abs

class EconomySetCommand : CommandManager("set", "setar", "difinir") {
    override fun command(sender: CommandSender, args: Array<String>) {
        if (args.size < 2) {
            sendUsage(sender)
            return
        }
        val player = FakePlayer(args[0])
        var valor = Extra.fromMoneyToDouble(args[1])
        valor = abs(valor)
        EduEconomyPlugin.instance.manager.setCoins(player, valor)
        if (args.size >= 3) {
            if (args[2] == "-msg") {
                return
            }
        }
        sender.sendMessage(
            EduEconomyPlugin.instance.message("money-set")
                .replace("%player", player.name)
                .replace("%amount", Extra.formatMoney(valor))
        )
        player
            .sendMessage(
                EduEconomyPlugin.instance.message("money-changed")
                    .replace("%player", sender.name)
            )
    }

    init {
        usage = "/money set <jogador> <quantidade>"
        description = "Definir saldo para o jogador"
    }
}